src/main/java/hudson/plugins/robot/RobotPublisher.java

```diff
@@ -281,14 +280,14 @@ public void perform(Run<?, ?> build, FilePath workspace, Launcher launcher, Task
 				logger.println(Messages.robot_publisher_done());
 				logger.println(Messages.robot_publisher_copying());
 
-				if (!getDisableCopyFilesToBuildDir()){					
+				if (!getDisableCopyFilesToBuildDir()) {
```

Comment:
disableCopyFilesToBuildDir is not really needed, it may be implicated by doing:
`if (!DEFAULT_ARCHIVE_DIR.equals(getArchiveDirName())) {`

Reply:
Feel free to change it. It's your PR. 😁